tests/NewCypress/cypress/e2e/regression/trusts/academiesInTrust.cy.ts

```diff
@@ -0,0 +1,89 @@
+import academiesInTrustPage from "../../../pages/trusts/academiesInTrustPage";
+
+describe("Testing the components of the Academies in this trust page", () => {
+
+    describe("Details tab", () => {
+        beforeEach(() => {
+            cy.login()
+            cy.visit('/trusts/academies/details?uid=5712')
+        });
+
+        it("Checks the correct AIT details page headers are present", () => {
```

Comment:
AIT is not a common acronym. Is it needed at all given that we're in "Testing the components of the Academies in this trust page" > "Details tab"?

Reply:
Removed AIT from all of the titles - My thinking was to differentiate from the stuff for the details page outside of academies and trusts but I think that's pretty redundant